app/configs/tempCrashesTable.ts

```diff
@@ -0,0 +1,57 @@
+import { tempCrashesListViewColumns } from "./tempCrashesListViewColumns";
+import { QueryConfig, FilterGroup } from "@/utils/queryBuilder";
+import { DEFAULT_QUERY_LIMIT } from "@/utils/constants";
+
+const columns = tempCrashesListViewColumns.map((col) => String(col.path));
+
+const tempCrashesListViewfilterCards: FilterGroup[] = [
```

Comment:
this is a bit of a hack, because we don't need to give user's the ability to switch **Temp records** on/off. i have a few ideas for this:

1. Add a `staticFilters` prop to the query config so that you can essentially hardcode extra filters that are not controlled through the UI
2. Move this whole feature into the main `/crashes` list and add a **Temporary records** filter to that page. Do we really need a separate page route just for creating these temp crashes?

Reply:
Yeah, this is interesting to think about since the hardcoded filters would be hidden from users which could be confusing if used in other places. But, I also see that being able to see non-temp crashes isn't really needed or useful in this view.

I like the idea of merging the two views/tables since this is essentially a duplicate table. Especially since one of the outcomes of the data model work was rolling the temp crashes into the crashes table and introducing the `is_temp_record` column. It would also subtly communicate that all totals are coming from CRIS crashes + "temp" crashes.